acceptance/lib/pxp-agent/config_helper.rb

```diff
@@ -201,6 +213,8 @@ def alt_ssl_cert_file(host, client_number)
     "#{ALT_SSL_BASE_DIR_WINDOWS}#{SSL_CERT_FILE_DIR_WINDOWS}client#{client_number}.alt.example.com.pem"
   when /osx/
     "#{ALT_SSL_BASE_DIR_OSX}#{SSL_CERT_FILE_DIR_POSIX}client#{client_number}.alt.example.com.pem"
+  when /solaris-10/
+    "#{ALT_SSL_BASE_DIR_SOL10}#{SSL_CERT_FILE_DIR_POSIX}client#{client_number}.alt.example.com.pem"
```

Comment:
I suspect we can tidy up the code a bit; we have 5 repetitive case statements in this file now.
Maybe by adding a method base_dir(host, use_alt_path=false) ?